total_station_node:
  ros__parameters:
    total_station_port: '/dev/cg_dev/tty_ts16'
    total_station_verbose: True
    positional_rms: 0.005
    total_station_hz: 10
    raw_topic: '/total_station/'
